### ISSUE TITLE
Will v1.0.6 be in the NVDA Add-On Store?

### ISSUE THREAD

**Issue by britechguy** [grisov/checkGestures#11]
I've downloaded it directly from here, and it is compatible with NVDA 2024.1, and I presume that was one of the main purposes for this dot level release.

It is not, at least at the moment, showing up in the Add-On Store, but the prior version, 1.0.5, is and is noted as incompatible.

I just wanted to make you aware of this in case there is a hitch somewhere preventing version 1.0.6 from making it into the Add-On Store.

**Comment by britechguy** [comment]
I'm giving this a bump, simply because we are now on the cusp of NVDA 2025.1, and NVDA core does not implement checking for input gesture collisions.  Until this functionality becomes part of the NVDA core, I think this add-on is of vital importance.  As such, if it is at all possible, it should be included in the NVDA Add-On store.  But that can only happen if an add-on's developer takes the steps to make that occur.

**Comment by grisov** [comment]
Dear Brian,

Thank you for your attention to this addon!

There have been a few user suggestions for improving its functionality that I have not implemented yet.

This weekend, I will try to publish this addon in the NVDA store in its current state.
I will start working on its improvements later.

All the best to you!

Oleksandr
